cfme/middleware/provider.py

```diff
@@ -101,15 +106,15 @@ def num_deployment(self):
 
     @num_deployment.variant('ui')
     def num_deployment_ui(self):
-        return int(self.get_detail("Relationships", "Middleware Deployments"))
+        return int(self.get_detail("Relationships", self.mv_deployments))
 
     @variable(alias='db')
     def num_server(self):
         return self._num_db_generic('middleware_servers')
 
     @num_server.variant('ui')
     def num_server_ui(self):
-        return int(self.get_detail("Relationships", "Middleware Servers"))
+        return int(self.get_detail("Relationships", self.mv_servers))
```

Comment:
ok, better solution, but what about existing codes where is used in this way?  other provider as well.